src/labels/CustomLabels.labels

```diff
@@ -1696,6 +1696,14 @@
         <protected>true</protected>
         <shortDescription>Error when user does not have a permission to change an RD originated in Elevate</shortDescription>
         <value>Recurring Donations from Elevate Payment Services can&apos;t be modified in NPSP except by the Elevate Payment Services Integration user.</value>
+    </labels>
+        <labels>
+        <fullName>RD2_ElevateRDAmountMustBeValid</fullName>
+        <categories>Recurring-Donations</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Validate Elevate Recurring Donations data if value on the Amount is valid or not</shortDescription>
+        <value>Enter a valid Amount. The Amount cannot be less than 1.</value>
```

Comment:
@mattwashington Please review the 2 new custom labels for invalid Amount and Effective Date

Reply:
**Amount**
Description
Checks if an Elevate Recurring Donation’s amount is valid or not

Value 
Please enter a valid Amount. The Amount cannot be less than 1.

**Effective Date**
Description 
Checks if Elevate Recurring Donations Effective Date is valid or not

Value
Please enter a new Effective Date.The Effective Date for an Elevate Recurring Donations cannot be in the future.